mne_qt_browser/_pg_figure.py

```diff
@@ -2780,22 +2780,20 @@ def __init__(self, **kwargs):
             self.mne.use_opengl = (
                     get_config('MNE_BROWSE_USE_OPENGL', '').lower() == 'true')
 
-        # Epochs currently only work with OpenGL enabled
+        # OpenGL needs to be enabled on macOS
         # (https://github.com/mne-tools/mne-qt-browser/issues/53)
-        mac_epochs = self.mne.is_epochs and sys.platform == 'darwin'
-        if mac_epochs:
+        if sys.platform == 'darwin':
             self.mne.use_opengl = True
 
         if self.mne.use_opengl:
             try:
                 import OpenGL
             except (ModuleNotFoundError, ImportError):
-                warn('PyOpenGL was not found and OpenGL can\'t be used!\n'
-                     'Consider installing pyopengl with pip or conda'
-                     'or set "use_opengl" to False to avoid this warning.')
-                if mac_epochs:
-                    warn('Plotting epochs on MacOS without OpenGL'
-                         'may be unstable!')
+                warn('PyOpenGL was not found and OpenGL cannot be used. '
+                     'Consider installing pyopengl with pip or conda or set '
+                     '"use_opengl=False" to avoid this warning.')
+                if sys.platform == 'darwin':
+                    warn('Plotting on macOS without OpenGL may be unstable!')
```

Comment:
I think we should raise an exception honestly, if we cannot even guarantee we're not going to crash!

Reply:
Fine with me. Which exception would you raise?

Reply:
re-raise the `ImportError`, maybe together with the warning text you have here

Reply:
Hm ok in that case, a clear warning might be okay. I'm just worried it might first get unnoticed and later become invisible because the segfault output fills up the scrollback buffer

Reply:
https://github.com/mne-tools/mne-qt-browser#development-and-testing says

> pytest -m benchmark mne_qt_browser

For me, it's faster for raw, slower for epochs, and sometimes causes errors. So I'd just leave it off on my system...

One option is to raise if the value is not passed (i.e., if `use_opengl=None` to start) *and* it's not set in the config var MNE_BROWSER_USE_OPENGL. That would make it so that a naive user will see an error, but an expert user who explicitly passes `use_opengl=False` or sets MNE_BROWSER_USE_OPENGL=false will not see an error. In these expert cases I actually wouldn't even bother to emit a warning.

Reply:
`E   FileNotFoundError: fname does not exist: /Users/clemens/Repositories/mne-qt-browser/.venv/lib/python3.10/site-packages/mne/io/tests/data/test_raw.fif`

How do I get this file?

And yes, I like your suggestion!

Reply:
If you `pip install -e .` or `python setup.py develop` your locally cloned MNE-Python it'll be there. We don't ship the testing files with the PyPi release, but they're there in the GitHub repo.

Reply:
So I need to clone `mne-python` inside `mne-qt-browser`? Can we put `pip install -U -e git+https://github.com/mne-tools/mne-python.git#egg=mne` in the development docs for future reference (in case it's not mentioned already)?

Reply:
There is absolutely *no* difference between `use_opengl=True` and `use_opengl=False` on my Mac.

Reply:
So about that behavior: I would like to make `use_opengl=True` the default on macOS. We warn if `pyopengl` is not installed and set `use_opengl=False`, but this case is unlikely to happen when we include `pyopengl` as a requirement on macOS.

In all other cases, it seems like at least it is beneficial when plotting epochs. But I really don't want to make any changes here that I haven't tested. I haven't used this browser at all, so I would like you to decide how to proceed.

Reply:
> I would like you to decide how to proceed.

My vote is for my proposal in https://github.com/mne-tools/mne-qt-browser/pull/63#issuecomment-1051052302 . Maybe it's just easier for me to push a commit to do it?

Reply:
Yes, please go ahead and make the changes!